<?xml version="1.0" encoding="UTF-8" ?>
<testsuites errors="1"
            failures="0"
            tests="1">

  <testsuite name="4.2test"
    errors="1" failures="0" hostname="" id="" package="4.2test" skipped="0"
    tests="1" time="0.004000" timestamp="Wed, 25 Jul 2018 03:30:51 GMT">
  
    <testcase name="search for valid city" classname="4.2test" time="0.004000" assertions="0">
    </testcase>
  

  
    <system-err>
        Error while running [4 2test / search for valid city]:

  Error: Search Again was not found in &#34;pageObject&#34;. Available elements: enterLoc,subBut,appTitle,searchField,weatherLoc,searchAgain,tryAgain,errorMess
    at getElement (/usr/local/lib/node_modules/nightwatch/lib/page-object/command-wrapper.js:13:13)
    at Page.waitForElementVisible (/usr/local/lib/node_modules/nightwatch/lib/page-object/command-wrapper.js:81:32)
    at Object.search for valid city (/Users/danielflanagan/devMtn/4.2project/tests/4.2test.js:23:10)
    at Module.call (/usr/local/lib/node_modules/nightwatch/lib/runner/module.js:62:34)
    at /usr/local/lib/node_modules/nightwatch/lib/runner/testcase.js:70:29
    at _fulfilled (/usr/local/lib/node_modules/nightwatch/node_modules/q/q.js:834:54)
    at self.promiseDispatch.done (/usr/local/lib/node_modules/nightwatch/node_modules/q/q.js:863:30)
    at Promise.promise.promiseDispatch (/usr/local/lib/node_modules/nightwatch/node_modules/q/q.js:796:13)
    at /usr/local/lib/node_modules/nightwatch/node_modules/q/q.js:556:49
    at runSingle (/usr/local/lib/node_modules/nightwatch/node_modules/q/q.js:137:13)
    </system-err>
  

  
  </testsuite>
</testsuites>
